source/fundamentals/collations.txt

```diff
@@ -4,5 +4,302 @@ Collations
+to ignore differences in letter accents.
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "fr", Strength: 1}
+   opts := options.CreateCollection().SetCollation(myCollation)
+   err := db.CreateCollection(context.TODO(), "books", opts)
+
+   if err != nil {
+      log.Fatal(err)
+   }
+
+Use the Default Collection Collation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you call an operation that supports collation on the ``books`` collection, the operation
+will now use the default collation. Assume you want to insert the following documents
+into ``books``:
+
+.. code-block:: json
+
+   {"name" : "Emma", "author" : "Austen", "length" : "474"}
+   {"name" : "Les Misérables", "author" : "Hugo", "length": "1462"}
+   {"name" : "Infinite Jest", "author" : "Wallace", "length" : "1104"}
+   {"name" : "Cryptonomicon", "author" : "Stephenson", "length" : "918"}
+   {"name" : "Ça", "author" : "King", "length" : "1138"}
+
+.. note::
+
+   To learn how to insert documents, see :ref:`golang-insert-guide`.
+   
+After inserting these documents, you can use an operation with the default collation
+on the ``books`` collection specified in the :ref:`golang-collection-collation`
+section. The following example uses the ``Find()`` method to return all documents with
+a ``name`` value that alphabetically precedes ``"Infinite Jest"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+      filter := bson.D{{"name", bson.D{{"$lt", "Infinite Jest"}}}}
+      cursor, err := coll.Find(context.TODO(), filter)
+
+      var results []bson.D
+      if err = cursor.All(context.TODO(), &results); err != nil {
+         panic(err)
+      }
+      for _, result := range results {
+         fmt.Println(result)
+      }
+
+   .. output::
+      :language: none
+      :visible: false
+
+      [{name Ça} {author King} {length 1138}]
+      [{name Cryptonomicon} {author Stephenson} {length 918}]
+      [{name Emma} {author Austen} {length 474}]
+
+Without specifying a default ``books`` collation, the ``Find()`` method would follow default
+binary collation rules to determine the ``name`` values that precede ``"Infinite Jest"``. These
+rules place words beginning with "Ç" after those beginning with "I". The output would resemble
+the following:
+
+.. code-block:: json
+   :copyable: false
+
+   [{name Cryptonomicon} {author Stephenson} {length 918}]
+   [{name Emma} {author Austen} {length 474}]
+
+See :ref:`golang-retrieve` for help with using the ``Find()`` method.
+
+Set a Collation on an Index 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After creating the ``books`` collection and specifying a collation, you cannot set a new default 
+collation directly. However, you can create an index with a new collation. 
+
+The following example uses the ``CreateOne()`` method to create an ascending index on the ``name``
+field and specifies a new collation with an ``"en_US"`` locale:
```

Comment:
Issue: I'm not sure the tenses of "create" and "specifies" agree. 

```suggestion
field and specify a new collation with an ``"en_US"`` locale:
```

Reply:
I left this as it is -- the tenses of "uses" (as in "uses the CreateOne() method" ...) and "specifies" match so I think it's good (lmk if you still disagree)